picard/ui/scripteditor.py

```diff
@@ -649,6 +654,8 @@ def eventFilter(self, object, event):
         evtype = event.type()
         if evtype in {QtCore.QEvent.WindowActivate, QtCore.QEvent.FocusIn}:
             self.update_examples()
+        elif object == self.ui.script_title and evtype == QtCore.QEvent.MouseButtonDblClick:
+            self.details_action.trigger()
```

Comment:
Should'nt we return `True` to prevent further event processing in this case?

https://doc.qt.io/qtforpython/overviews/eventsandfilters.html#event-filters

Reply:
I think yes. It probably has no real effect here, because there is not other default action on double click, but still would be the proper way to do this to avoid potential other effects.